src/Generation/GapicClientGenerator.php

```diff
@@ -48,6 +48,10 @@
 
 class GapicClientGenerator
 {
+    // This is the variable name for the callOptions(prev optionalArgs)
+    // in the GAPIC clients.
+    const CALL_OPTIONS_VAR = 'callOptions';
```

Comment:
nit: make this a private constant
```suggestion
    private const CALL_OPTIONS_VAR = 'callOptions';
```